psycopg2cffi/_impl/connection.py

```diff
@@ -121,7 +121,13 @@ def __init__(self, dsn, async=False):
         # The number of commits/rollbacks done so far
         self._mark = 0
 
-        self._async = async
+        if 'async' in kwargs:
+            self._async = kwargs.pop('async')
+        if 'async_' in kwargs:
+            self._async = kwargs.pop('async_')
+        else:
+            self._async = False
```

Comment:
Will be also nice to check that kwargs are empty after we determine ``self._async`` value, else unsupported ``kwargs`` will be silently ignored instead of rising the error (e.g. ``Connection(do_what_i_want=True)`` will not raise an error after this change).